source/glbinding/CMakeLists.txt

```diff
@@ -5,6 +5,22 @@
 
 find_package(OpenGL REQUIRED)
 
+set(ADDITIONAL_LIBRARIES)
+set(ADDITIONAL_INCLUDES)
+if(OPTION_BUILD_WITH_BOOST_THREAD)
+    find_package(Boost COMPONENTS thread REQUIRED)
+    
+    if (Boost_FOUND)
+        message(STATUS "Use Boost for thread.")
+        
+        set(ADDITIONAL_LIBRARIES ${ADDITIONAL_LIBRARIES} ${Boost_LIBRARIES})
+        set(ADDITIONAL_INCLUDES ${ADDITIONAL_INCLUDES} ${Boost_INCLUDE_DIRS})
+    else()
+        message(WARNING "OPTION_BUILD_WITH_BOOST_THREAD is set to On: Boost not found.")
+        message(WARNING "Defaulting to C++11 thread.")
```

Comment:
Is this accurate? On line 278, it looks like `OPTION_BUILD_WITH_BOOST_THREAD` defines `GLBINDING_USE_BOOST_THREAD` regardless of whether Boost was found.